webapp/templates/base.html.twig

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8">
-        <title>{% block title %}Welcome!{% endblock %}</title>
-        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text></svg>">
-        
-        <link rel="preload" as="style" href="/styles/app.css" />
-        <link rel="stylesheet" href="/styles/app.css" />
-
-        {% block stylesheets %}
-            {{ encore_entry_link_tags('app') }}
-        {% endblock %}
-
-        {% block javascripts %}
-            {{ encore_entry_script_tags('app') }}
-        {% endblock %}
-    </head>
-    <body>
-        {% block body %}{% endblock %}
-    </body>
+	<head>
+		<meta charset="UTF-8">
+		{% block meta %}{% endblock %}
+		<title>
+			{% block title %}{% endblock %}
+		</title>
+		<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text></svg>">
+
+		<link rel="preload" as="style" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"/>
+		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+		<link rel="preload" as="style" href="/styles/app.css"/>
+		<link rel="stylesheet" href="/styles/app.css"/>
+
+		{% block stylesheets %}
+			{{ encore_entry_link_tags('app') }}
+		{% endblock %}
+
+		{% block javascripts %}
+			{{ encore_entry_script_tags('app') }}
+		{% endblock %}
+	</head>
+	<body>
+		<header>
+			<a href="{{ url('home') }}">Home</a>
+
+			{% set player = app.session.get('player') %}
+			{% if player == NULL %}
+				<form action="{{ url('player_sign_in') }}" method="POST">
+					<input type="hidden" name="sign_in[_token]" value="{{ csrf_token('sign_in') }} ">
+
+					<input type="text" name="sign_in[name]" id="sign-in-name" placeholder="Username">
+
+					<button type="submit">Sign In</button>
```

Comment:
```suggestion
					<button type="submit">Sign In / Sign Up</button>
```

Since, if I'm not mistaken, you can actually do both depending on whether or not the username already exists